src/css/pages/maintenance.css

```diff
@@ -0,0 +1,68 @@
+.slds-illustration {
+  & .slds-illustration__stroke-primary {
+    stroke: var(--slds-g-color-palette-cloud-blue-80,#90d0fe);
+  }
+
+  & .slds-illustration__stroke-secondary {
+    stroke: var(--slds-g-color-palette-cloud-blue-90,#cfe9fe);
+  }
+
+  & .slds-illustration__fill-primary {
+    fill: var(--slds-g-color-palette-cloud-blue-80,#90d0fe);
+  }
+
+  & .slds-illustration__fill-secondary {
+    fill: var(--slds-g-color-palette-cloud-blue-90,#cfe9fe);
+  }
+
+  & div {
+    margin: auto;
+    text-align: center;
+  }
```

Comment:
Can move the following shared 2 properties into here:

```css
    text-align: center;
    word-break: keep-all;
```

and then delete the properties from .slds-text-title and .slds-text-description.